addons/gift_card/views/views.xml

```diff
@@ -39,25 +32,17 @@
                 <sheet>
                     <group>
                         <group>
-                            <field name="buy_line_id" string="Sale Order"/>
```

Comment:
label missing in the move :/ It is why we have test from PO to improve usability.
'Buy line' for end user is not clear.